app/src/main/res/values/bridges_strings.xml

```diff
@@ -50,9 +50,10 @@
     <string name="bridge_usdc_usdt_swap_action">Swap USDC → USDCet</string>
     <string name="bridge_how_to_claim_for_free_title">How to claim for free</string>
     <string name="bridge_claim_description_progress">It usually takes 15–20 minutes for a transaction to complete</string>
-    <string name="bridge_ethereum_network">Ethereum network</string>
-    <string name="bridge_claim_pending">Claiming…</string>
-    <string name="bridge_send_pending">Sending…</string>
+    <string name="bridge_to_ethereum">To Ethereum</string>
```

Comment:
i've already added this
transaction_history_wh_send

Reply:
done in PWN-8397 - use title and subtitle from designs
like others
check commit pls

Reply:
i mean we need to remove duplicates
my string or your
and use it from one source